webpack.config.js

```diff
@@ -1,9 +1,28 @@
 var path = require('path');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
 
 module.exports = {
-  entry: "./public/javascripts/app.js",
+  entry: "./client/app.js",
```

Comment:
it is better to specify `context: path.join(__dirname, "client")` here and use path relative to it for entry